api/routers/areas.py

```diff
@@ -31,11 +32,34 @@ async def list_areas():
     }
 
 
+def get_all_cameras_ids():
+    ids_list = [camera['id'] for camera in get_cameras()]
+    return ",".join(ids_list)
+
+
+def all_cameras_area():
+    # Returns information about all the cameras in one area.
+    return {
+        "violation_threshold": -1,
+        "notify_every_minutes": -1,
+        "emails": "",
+        "enable_slack_notifications": False,  # "N/A"
+        "daily_report": False,  # "N/A"
+        "daily_report_time": "N/A",
+        "occupancy_threshold": -1,
+        "id": "All",
```

Comment:
Maybe we can use here the constant `ALL_AREAS`

Reply:
Great!